### PR TITLE
gui: Display device name in modal

### PR DESCRIPTION
**Pull request by Shidory**
Edit device name in modal

### CODE REVIEW

**Comment by calmh** [comment]
Hi!

Thanks for the contribution! This is a good start, but we should also have the same for edited folders and not just edited devices. As for the formatting, I'd suggest not adding the colon in the translated string, instead just adding the device in `()` parenthesis when it's there. Also you might want to use the `deviceName()` function which grabs the name when there is one and otherwise shows the short device ID.

**Comment by Shidory** [comment]
Oh, thanks. I'll see to it.

**Comment by uok** [comment]
Thanks, this fixes #5380

File: gui/default/syncthing/device/editDeviceModalView.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<modal id="editDevice" status="default" icon="{{editingExisting ? 'fas fa-pencil-alt' : 'fas fa-desktop'}}" heading="{{editingExisting ? 'Edit Device' : 'Add Device' | translate}}" large="yes" closeable="yes">
+<modal id="editDevice" status="default" icon="{{editingExisting ? 'fas fa-pencil-alt' : 'fas fa-desktop'}}" heading="{{editingExisting ? 'Edit Device : ' : 'Add Device' | translate}} {{currentDevice.name}}" large="yes" closeable="yes">
```
Comment (calmh) [inline gui/default/syncthing/device/editDeviceModalView.html]:
```suggestion
<modal id="editDevice" status="default" icon="{{editingExisting ? 'fas fa-pencil-alt' : 'fas fa-desktop'}}" heading="{{editingExisting ? 'Edit Device' : 'Add Device' | translate}} {{currentDevice.name}}" large="yes" closeable="yes">
```

**Pull request merged**